packages/client/index.js

```diff
@@ -221,7 +221,7 @@ async function buildCallFunction (spec, baseUrl, path, method, methodMeta, throw
     }
 
     if (telemetryId) {
-      headers['x-telemetry-id'] = telemetryId
+      headers['x-platformatic-telemetry-id'] = telemetryId
```

Comment:
maybe use plt as a prefix instead? It's shorter... less bytes to move?

Reply:
done